src/run/emulator.rs

```diff
@@ -1654,6 +1697,88 @@ impl Machine {
                             Err(ExecutionError::new("invalid operand to EcPairing instruction", &self.state, None))
                         }
 
+                    }
+                    Opcode::AVMOpcode(AVMOpcode::NewBuffer) => {
+                        let _size = self.stack.pop_uint(&self.state)?;
```

Comment:
What is the purpose of the size arg to this instruction?  It seems to be unused.

(In the original ByteArray implementation in the ArbOS library, bytearray_new took a uint arg that was intended as a size hint.)